internal/pkg/plugin/trellogithub/helper.go

```diff
@@ -1,62 +1,23 @@
 package trellogithub
 
 import (
-	"bytes"
 	"fmt"
-	"text/template"
-
-	"github.com/mitchellh/mapstructure"
-
-	"github.com/merico-dev/stream/pkg/util/github"
-	"github.com/merico-dev/stream/pkg/util/trello"
 )
 
-// renderTemplate render the github actions template with config.yaml
-func (tg *TrelloGithub) renderTemplate(workflow *github.Workflow) error {
-	var jobs Jobs
-	err := mapstructure.Decode(tg.options.Jobs, &jobs)
-	if err != nil {
-		return err
-	}
-	//if use default {{.}}, it will confict (github actions vars also use them)
-	t, err := template.New("trellogithub").Delims("[[", "]]").Parse(workflow.WorkflowContent)
-	if err != nil {
-		return err
-	}
-
-	var buff bytes.Buffer
-	err = t.Execute(&buff, jobs)
-	if err != nil {
-		return err
-	}
-	workflow.WorkflowContent = buff.String()
-	return nil
-}
-
 func buildState(tg *TrelloGithub, ti *TrelloItemId) map[string]interface{} {
```

Comment:
The `ti` param is not used since the `ti.xxx` has been deleted. so it should be removed.